ci/docker/buildenv/build-systemc.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+build_variant () {
+    version=`$1 --version | grep gcc | awk '{print $4}'`
+    mkdir /usr/local/systemc-2.3.3-gcc$version
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/manual/wsnyder/verilator/pullRequest?prid=4756540)